src/CodeStyle/Core/CodeFixes/FormattingCodeFixProvider.cs

```diff
@@ -77,14 +77,14 @@ public FixAll(AbstractFormattingCodeFixProvider formattingCodeFixProvider)
                 _formattingCodeFixProvider = formattingCodeFixProvider;
             }
 
-            protected override string CodeActionTitle => CodeStyleResources.Fix_formatting;
+            protected override string GetCodeActionTitle(FixAllContext fixAllContext) => CodeStyleResources.Fix_formatting;
```

Comment:
Why this change?

Reply:
I made this change before moving SyntaxEditorBasedCodeFixProvider to shared layer. No longer required, so reverted all changes to this file.